sdk/communication/Azure.Communication.NetworkTraversal/tests/CommunicationRelayClient/CommunicationRelayClientLiveTests.cs

```diff
@@ -60,6 +60,40 @@ public async Task GettingTurnCredentialsGeneratesTurnCredentials(AuthMethod auth
         }
 
         [Test]
+        [TestCase(AuthMethod.ConnectionString, TestName = "GettingTurnCredentialsWithConnectionStringRouteType")]
+        [TestCase(AuthMethod.KeyCredential, TestName = "GettingTurnCredentialsWithKeyCredentialRouteType")]
+        [TestCase(AuthMethod.TokenCredential, TestName = "GettingTurnCredentialsWithTokenCredentialRouteType")]
+        public async Task GettingTurnCredentialsGeneratesTurnCredentialsWithNearestRouteType(AuthMethod authMethod, params string[] scopes)
```

Comment:
Since it is a readOnly struct RouteType I can't add it in the list, I tried but it says that RouteType is not constant, type or array. But I'm adding the test case

Reply:
Added them but there's an issue with the type since it's a readonly struct. It asks to be a constant, type or array. So Just added a new test cases.